src/signal.rs

```diff
@@ -382,14 +382,24 @@ mod tests {
     use std::thread;
     use std::time::Duration;
 
+    // Reserve for each vcpu signal.
     static mut SIGNAL_HANDLER_CALLED: bool = false;
 
     extern "C" fn handle_signal(_: c_int, _: *mut siginfo_t, _: *mut c_void) {
         unsafe {
+            // In the tests, there only uses vcpu signal.
             SIGNAL_HANDLER_CALLED = true;
         }
     }
 
+    fn is_pending(signal: c_int) -> bool {
```

Comment:
This looks useful enough to become a pub function in the main `signal.rs` module. Looks good otherwise.

Reply:
Thanks for your review. I think the function is_pending would not currently been used by other codes so we could move it in the future once it’s needed. WDYT?